Gems/Atom/RHI/Vulkan/Code/Source/Platform/Mac/Vulkan_Traits_Mac.h

```diff
@@ -8,8 +8,10 @@
 #pragma once
 
 #define AZ_TRAIT_ATOM_SHADERBUILDER_DXC "Builders/DirectXShaderCompiler/bin/dxc"
-#define AZ_TRAIT_ATOM_AZSL_SHADER_HEADER UNUSED_TRAIT // Mac can not build vulkan shaders at the moment
-#define AZ_TRAIT_ATOM_AZSL_PLATFORM_HEADER UNUSED_TRAIT // Mac can not build vulkan shaders at the moment
+#define AZ_TRAIT_ATOM_AZSL_SHADER_HEADER "" // Mac can not build vulkan shaders at the moment
+#define AZ_TRAIT_ATOM_AZSL_PLATFORM_HEADER "" // Mac can not build vulkan shaders at the moment
+#define AZ_TRAIT_ATOM_MOBILE_AZSL_SHADER_HEADER ""
```

Comment:
Extend the comment to these two as well
// Mac can not build vulkan shaders at the moment